SDK v2 migration guide.md

```diff
@@ -1,13 +1,13 @@
-# SDK version 2.x migration guide
+# SDK version 2 migration guide
 
-This document outlines the changes made from this library's 1.x releases to its 2.x releases. Since this is
+This document outlines the changes made from this library's 1 releases to its 2 releases. Since this is
 a major version upgrade, there are a number of breaking changes that will affect the ability to compile. Provided here
-are outlines of the notable breaking changes as well as a mapping from version 1.x APIs to version 2.x APIs to aid migrating.
+are outlines of the notable breaking changes as well as a mapping from version 1 APIs to version 2 APIs to aid migrating.
 
 ## Table of contents
 
- - [Why the version 1.x SDK is being replaced](#Why-the-version-1.x-sdk-is-being-replaced)
- - [What will happen to the version 1.x SDK](#What-will-happen-to-the-Version 1.x-sdk)
+ - [Why the version 1 SDK is being replaced](#why-the-version-1-sdk-is-being-replaced)
```

Comment:
This and next link were broken. Dashes not spaces. All lowercase. The dot (in 1.x) seemed to break it, so I just removed it. We don't need to be so specific for version number.